src/fidesops/ops/core/config.py

```diff
@@ -167,12 +174,19 @@ class FidesopsConfig(FidesSettings):
     database: FidesopsDatabaseSettings
     redis: RedisSettings
     security: FidesopsSecuritySettings
-    execution: ExecutionSettings
-    root_user: RootUserSettings
-    admin_ui: AdminUiSettings
-    notifications: FidesopsNotificationSettings
-
-    port: int
+    execution: Optional[ExecutionSettings] = ExecutionSettings()
+    root_user: Optional[RootUserSettings] = RootUserSettings()
+    admin_ui: Optional[AdminUiSettings] = AdminUiSettings()
+    notifications: Optional[
+        FidesopsNotificationSettings
+    ] = FidesopsNotificationSettings()
+
+    port: int = int(
```

Comment:
I'd be open to other ways of doing this than casting to `int(...)` though it seemed practical for now.

My main gripe is that this could be confusing for users as the env vars are expected to be strings, where port must then be an integer. It's technically possible to run Fidesops with `FIDESOPS__PORT="iamastring"` and the error message would be along the lines of `ValueError: invalid literal for int() with base 10: 'iamastring'`.